web/cypress/e2e/auto-tax-filings.spec.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import {
+  clickModalSaveButton,
+  completeBusinessStructureTask,
+  openFormationDateModal,
+  selectDate,
+  selectLocation,
+} from "@businessnjgovnavigator/cypress/support/helpers/helpers";
+import { completeNewBusinessOnboarding } from "@businessnjgovnavigator/cypress/support/helpers/helpers-onboarding";
+import { onProfilePage } from "@businessnjgovnavigator/cypress/support/page_objects/profilePage";
+import { LookupIndustryById } from "@businessnjgovnavigator/shared/";
+import { onDashboardPage } from "cypress/support/page_objects/dashboardPage";
+
+describe("auto tax filing [feature] [all] [group4]", () => {
+  let businessName: string;
+  beforeEach(() => {
+    cy.loginByCognitoApi();
+    const industry = LookupIndustryById("cosmetology");
```

Comment:
[sand] Does the industry matter? Could we make this random?

Reply:
Absolutely!